code/game/objects/structures/window.dm

```diff
@@ -446,6 +446,9 @@
 /obj/structure/window/spawner/north
 	dir = NORTH
 
+/obj/structure/window/spawner/south
+	dir = SOUTH
+
```

Comment:
we should really change these over to be `MAPPING_DIRECTIONAL_HELPERS()` with an offset of 0.